lib/lbt/bundle/Builder.js

```diff
@@ -40,54 +41,88 @@ function isEmptyBundle(resolvedBundle) {
 	return resolvedBundle.sections.every((section) => section.modules.length === 0);
 }
 
-const EVOBundleFormat = {
-	beforePreloads(section) {
+class BundleBuilder {
+	constructor(pool, targetUi5Version) {
+		this.pool = pool;
+		this.resolver = new BundleResolver(pool);
+		this.splitter = new BundleSplitter(pool, this.resolver);
+		this.targetBundleFormat = null;
+		this.targetUi5Version = targetUi5Version;
+	}
+
+	getUi5MajorVersion() {
+		if (this.targetUi5VersionMajor !== undefined) {
+			return this.targetUi5VersionMajor;
```

Comment:
maybe initialise `this.targetUi5VersionMajor` in the constructor as `undefined` so that it's clear the variable exists?

Reply:
true. done